src/common/components/help/help-install-snap.js

```diff
@@ -25,15 +25,15 @@ export default class HelpInstallSnap extends Component {
           </p>
         }
         <p className={ styles.p }>
-          Don’t have snapd installed?  {' '}
+          (
```

Comment:
I don't understand, the spec has `(Don’t have snapd installed?)` with the text linked and the parens unlinked.